src/Controller/ExportController.php

```diff
@@ -25,7 +25,6 @@
 use Shopware\Core\Framework\DataAbstractionLayer\Search\EntitySearchResult;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\IdSearchResult;
-use Shopware\Core\Framework\Routing\Annotation\RouteScope;
```

Comment:
This should not be removed, as it used in a Symfony annotation. Please install the Symfony Plugin for your IDE. With it this won't count as "not imported". Also the tests break due to this :).

Reply:
After Symfony plugin was installed, nothing seems unused according to IDE so I just reverted the last commit.